frontend/jupyter/src/app/pages/form/form-default/volume/mount/mount.component.ts

```diff
@@ -1,15 +1,39 @@
-import { Component, Input, OnInit } from '@angular/core';
-import { FormGroup } from '@angular/forms';
+import { Component, Input, OnChanges, OnDestroy, OnInit, SimpleChanges } from '@angular/core';
+import { FormGroup, Validators } from '@angular/forms';
 
 @Component({
   selector: 'app-volume-mount',
   templateUrl: './mount.component.html',
   styleUrls: ['./mount.component.scss'],
 })
-export class VolumeMountComponent implements OnInit {
+export class VolumeMountComponent implements OnInit, OnChanges {
   @Input() volGroup: FormGroup;
 
   constructor() {}
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+    this.initComponent();
+  }
+  ngOnChanges(): void {
+    this.initComponent();
```

Comment:
Is this needed to be also in the onChanges function? Shouldnt it be enough to just be in the onInit function?

Reply:
It is to force the validation when text is changed.